### ISSUE TITLE
php-user-authentication reference

### ISSUE THREAD

**Issue by vladimirprieto** [SinghDigamber/php-user-authentication#2]
i think all references to http://localhost:8888/php-user-authentication should be removed.

when testing the code in your own machine, makes no sense to point to that URL

**Comment by PhillipRSA** [comment]
I agree, lucky I saw it and Shared Hosting Providers are NOT willing to open Port
I am not sure what the purpose of Port:8888 is
